src/test/utils/gameManager.ts

```diff
@@ -335,4 +335,23 @@ export default class GameManager {
     const modifier = candidate.type.newModifier(party[pokemonIndex]);
     this.scene.addModifier(modifier, false);
   }
+
+  /**
+   * Select a pokemon from the party menu. Only really handles the basic cases
+   * of the party UI, where you just need to navigate to a party slot and press
+   * Action twice - navigating any menus that come up after you select a party member
+   * is not supported.
+   * @param slot the index of the pokemon in your party to switch to
+   * @param inPhase Which phase to expect the selection to occur in. Typically
+   * non-command switch actions happen in SwitchPhase.
+   */
+  doSelectPartyPokemon(slot: integer, inPhase = "SwitchPhase") {
```

Comment:
```suggestion
  doSelectPartyPokemon(slot: number, inPhase = "SwitchPhase") {
```
moving forward, we want to replace all `integer` types with `number`

Reply:
oh, fair. usually I would number but I think I saw integer somewhere and was like "well when in rome" 😄